gulp/config.js

```diff
@@ -1,16 +1,7 @@
 'use strict';
 
 var fs = require( 'fs' );
-
-/**
- * Set up file paths
- */
-var loc = {
-  src:  './src',
-  dist: './dist',
-  lib:  JSON.parse( fs.readFileSync( './.bowerrc' ) ).directory, // eslint-disable-line no-sync, no-inline-comments, max-len
-  test: './test'
-};
+var paths = require( '../config/environment-production' ).paths;
```

Comment:
Thinking about it further, environment settings we currently have would be the same on production vs dev, but the webpack config would be different. I think I'll just remove `-production` for now.